chevah/python-package
# Zlib inflate security fix.

Scope
=====

Fixes CVE-2022-37434.


Changes
=======

Updated zlib sources to 1.2.13.

**Note** that Windows packages use upstream Python which embeds zlib 1.2.11, so they remain vulnerable to this.

Other updates: OpenSSL 1.1.1s, SQLite 3.39.4, libffi 3.4.4, `psutil` 5.9.3 (5.9.1 on generic glibc-based Linux).


How to try and test the changes
===============================

reviewers: @adiroiban 

To review changes:
```
git diff master python-modules/ chevah_build
```

Check automated tests.

Staging packages tested in server repo: https://github.com/chevah/server/pull/5974
